thrift/src/test/java/com/linecorp/armeria/common/thrift/text/CurrentThriftMessageClassFinderTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.thrift.text;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import com.linecorp.armeria.common.thrift.text.RpcDebugService.doDebug_args;
+import com.linecorp.armeria.common.util.SystemInfo;
+
+class CurrentThriftMessageClassFinderTest {
+
+    @ParameterizedTest(name = "testThriftMessageClassFinder {index}: finder={0}, isLookupCase={1}")
+    @MethodSource("testThriftMessageClassParameters")
```

Comment:
is it fine using MethodSource, unlike other codes? 
other codes are using ArgumentsSource.

Reply:
I think it's fine, but we don't need the second parameter - we can just write two tests, because they are testing two different scenarios.